src/dialect.js

```diff
@@ -185,26 +190,23 @@ class Dialect {
   _defaultBuilders() {
     return {
       'function': function (operator, parts) {
-        var operator = operator.substr(0, operator.length - 2).toUpperCase();
-        return operator + '(' + parts.join(', ') + ')';
+        const op = operator.substr(0, operator.length - 2).toUpperCase();
+        return op + `(${parts.join(', ')})`;
```

Comment:
does this need to be a template string? `.join` returns a string

Reply:
it adds the surrounding parens...